src/types/array.ts

```diff
@@ -1,9 +1,10 @@
-import {STRules} from '../rules';
 import {StrongType, makeStrong} from '../strong-type';
 
+import {STRules} from '../rules';
+
 export type StrongArray<T> = StrongType<T[]>;
 
-export function makeArray<T>(fallback: T[], initial?: T[] | null): StrongArray<T> {
+export function makeArray<T>(fallback: T[], initial: T[] | null | undefined): StrongArray<T> {
```

Comment:
- [ ] Change `| undefined` type on `initial` to use the optional operator instead `?`.